src/content/docs/release-notes/docs-release-notes/docs-6-14-2024.mdx

```diff
@@ -0,0 +1,85 @@
+
+### Major changes
+
+* We've moved away from the `applied intelligence` name, because it's confusing when compared to our other AI efforts. We completed a massive sweep to remove nearly all mention of "applied intelligence" from our docs.
+* Deleted several old, outdated alerts docs.
+* A rewrite of [change vulnerability status](/docs/vulnerability-management/change-vulnerability-status) makes it easier for you to enable this feature.
+
+### Minor changes
+
+* Removed the notifications section from [CodeStream settings](/docs/codestream/codestream-settings/account-settings).
+* Added a new OpenTelemetry [redis instance](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources) section to clarify that you need to manually include redis attributes for the collector.
+* Private Synthetics Job Managers now support [permanent data storage](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration/#permanent-data-storage) via synthetic monitoring.
+* Added New Relic partner API endpoint to networks [user-facing domains](/docs/new-relic-solutions/get-started/networks/#user-facing-domains) table.
+* Added version tracking metadata environment variables to the [Ruby agent's sending handled errors](/docs/apm/agents/ruby-agent/api-guides/sending-handled-errors-new-relic/#version-tracking).
+* Added environment variables for .NET Lambda monitoring for [serverless](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/configure-serverless-monitoring-aws-lambda/#dotnet) and [layerless](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-layerless/#dotnet).
+* Multiple docs updates related to hybrid mobile agent releases.
+* We've updated our docs to remove unsupported [.NET Core](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core) versions. (We no longer support .NET Core versions 2.0, 2.1, 2.2, and 3.0.)
+* In response to customer feedback, updated info on how to [view script logs](/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results/#viewing-logs) in the synthetic monitoring UI.
+* Thanks to our Docs engineering team, we no longer hardcode `v` into our release notes titles.
+* Quality-of-life improvements to [view slow query details](/docs/apm/apm-ui-pages/monitoring/view-slow-query-details) removed some outdated content and brought procedural content up-to-date.
+
+### Release notes and What's New posts
+
+* What's New post for [streamline log parsing](/whats-new/2024/06/whats-new-06-11-click-to-parse).
+* [Capacitor agent v1.4.0](src/content/docs/release-notes/mobile-release-notes/capacitor-release-notes/capacitor-agent-140)
```

Comment:
This might be silly, but in our previous section we mentioned removing `v` from release notes title. Does that mean we don't use lower case `v` at all anymore? I checked our style guide and it says we can, but I'm not sure what implications the eng update has for the style guide

Reply:
It's fine here. For release notes, we were literally hardcoding v into the release notes title. Which led to a funny thing where Docs release notes: version [date] had to be Docs release notes: ersion [date] or would show up as vversion